src/map/atcommand.cpp

```diff
@@ -5387,26 +5387,26 @@ ACMD_FUNC(jailtime){
 
 	nullpo_retr(-1, sd);
 
-	if (!sd->sc.data[SC_JAILED]) {
+	if (!sd->sc.getSCE(SC_JAILED)) {
```

Comment:
Here as well. The whole function accesses SC_JAILED multiple times in different ways.